### DISCUSSION TITLE
摘要，目录，参考文献，谢辞无需加粗，请问如何修改

### DISCUSSION THREAD

**Discussion by gxf0913** [sjtug/SJTUThesis#655] [提问] [answered]
摘要，目录，参考文献，谢辞无需加粗，请问如何修改？目录、参考文献和谢辞貌似用的是chapter的style

**Comment by skyzh** [comment, marked as answer]
在导言区可以设置相关的内容。SJTUThesis 默认的设置如下，可以把里面的 bfseries 去掉。

```latex
\ctexset{
  chapter = {
    format = \zihao{3}\bfseries\heiti\centering,
  }
}
```

**Reply by gxf0913** [reply]
但是chapter是需要加粗的

**Reply by skyzh** [reply]
本科毕设应该没有这个要求。可以考虑在取消全局加粗同时，给需要加粗的 chapter 加 \bfseries？

**Reply by gxf0913** [reply]
好的我试试，谢谢

**Reply by AlexaraWu** [reply]
《撰写规范》里都没有要求加粗，所以不要把 Word 当标准呀。:joy:
